docs/classes/checkout.orders.html

```diff
@@ -97,7 +97,7 @@ <h1><a href="https://developer.salesforce.com/docs/commerce/commerce-api/referen
 clientConfig.headers[<span class="hljs-string">&#x27;authorization&#x27;</span>] = <span class="hljs-string">`Bearer <span class="hljs-subst">${token.access_token}</span>`</span>;
 <span class="hljs-keyword">const</span> ordersClient = <span class="hljs-keyword">new</span> Checkout.Orders(clientConfig);</code></pre>
 					<span style="font-size:.7em; display:block; text-align: right">
-						API Version: 1.4.2<br />
+						API Version: 1.4.4<br />
```

Comment:
FYI all of the generated docs are already done with the latest downloaded APIs, even though they're in a separate PR.